tardis/plasma/properties/atomic.py

```diff
@@ -94,18 +98,14 @@ class PhotoIonizationData(ProcessingPlasmaProperty):
     """
     Attributes
     ----------
-    photo_ion_cross_sections : Pandas DataFrame (nu, x_sect,
-                                                index=['atomic_number',
-                                                       'ion_number',
-                                                       'level_number']),
-                                                dtype float)
-        Table of photoionization cross sections as a
-        function of frequency.
-    photo_ion_block_references : One-dimensional Numpy Array, dtype int
+    photo_ion_cross_sections : pandas.DataFrame, dtype float
+        Photoionization cross sections as a function of frequency.
+        nu, x_sect, index=['atomic_number','ion_number','level_number'].
```

Comment:
```suggestion
        Columns are nu, x_sect, index=('atomic_number','ion_number','level_number')
```
Index is multi-index here so we represent it as tuple braces